articles/data-factory/concepts-roles-permissions.md

```diff
@@ -22,6 +22,8 @@ To create Data Factory instances, the user account that you use to sign in to Az
 
 To create and manage child resources for Data Factory - including datasets, linked services, pipelines, triggers, and integration runtimes - the following requirements are applicable:
 - To create and manage child resources in the Azure portal, you must belong to the **Data Factory Contributor** role at the **Resource Group** level or above.
+  > [!NOTE]
+  > If you already have the **Contributor** role at the **Resource Group** level or above, you do not need the **Data Factory Contributor** role. The [Contributor role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#contributor) is a superset role that includes all permissions granted to the [Data Factory Contributor role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#data-factory-contributor).
```

Comment:
Let's add a line break space, and adjust the links to point to the relative file names.. 

```suggestion

  > [!NOTE]
  > If you already have the **Contributor** role at the **Resource Group** level or above, you do not need the **Data Factory Contributor** role. The [Contributor role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#contributor) is a superset role that includes all permissions granted to the [Data Factory Contributor role](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles#data-factory-contributor).
```